src/util/PathUtil.cpp

```diff
@@ -189,7 +189,7 @@ auto Util::getGettextFilepath(const char* localeDir) -> fs::path {
     }
     const char* dir = (gettextEnv) ? directories.c_str() : localeDir;
     g_debug("TEXTDOMAINDIR = %s, Platform-specific locale dir = %s, chosen directory = %s", gettextEnv, localeDir, dir);
-    return fs::path(dir);
+    return fs::u8path(dir);
```

Comment:
This is generally wrong, since gettextEnv has the system encoding. So we can't just expect utf-8.
See #5640 for a correct fix.

Reply:
After your test in https://github.com/xournalpp/xournalpp/pull/5640 this is correct, but the function and it's usage was odd in any way. It should accept a fs::path and not a string of unknown encoding.